src/Security/MemberAuthenticator/MemberLoginForm.php

```diff
@@ -131,23 +135,25 @@ protected function getFormFields()
             $backURL = $request->getSession()->get('BackURL');
         }
 
-        $label = Member::singleton()->fieldLabel(Member::config()->get('unique_identifier_field'));
+        $uniqueIdentifierFieldName = Member::config()->get('unique_identifier_field'); 
+        $label = Member::singleton()->fieldLabel($uniqueIdentifierFieldName);
+        
         $fields = FieldList::create(
             HiddenField::create("AuthenticationMethod", null, $this->getAuthenticatorClass(), $this),
             // Regardless of what the unique identifer field is (usually 'Email'), it will be held in the
             // 'Email' value, below:
             // @todo Rename the field to a more generic covering name
-            $emailField = TextField::create("Email", $label, null, null, $this),
+            $uniqueIdentifierField = ($uniqueIdentifierFieldName == 'Email' && !empty(DefaultAdminService::getDefaultAdminEmail())) ? EmailField::create("Email", $label, null, null, $this) : TextField::create($uniqueIdentifierFieldName, $label, null, null, $this), 
```

Comment:
This must remain a text field otherwise the login field will break for every site out there that does not use emails for their default admin user

Reply:
it would be a EmailField only when SS_DEFAULT_ADMIN_EMAIL is set, otherwhise it would be a TextField

Reply:
Why is the login field determined by the admin's type? What if that is different to normal members?